recipes/librhash/all/CMakeLists.txt

```diff
@@ -0,0 +1,102 @@
+cmake_minimum_required(VERSION 3.1.2)
```

Comment:
Why this migration to a custom CMake file to build this library? Shouldn't we add a comment here about it? Would users be concerned of this change as something critical?

Reply:
librhash provides a configure script that works find on Linux.
For Visual Studio, it provides a [vcxproj file](https://github.com/rhash/RHash/blob/d22963a90b960254208d2a57be4052644af69195/build/vc-2010/rhash.vcxproj#L5), that does enable setting the Visual Studio compiler runtime or shared/static library option.
The vcxproj itself looks very generic.

Reply:
If this library doesn't support visual studio or doesn't support shared or specific runtime, I would make those configurations as invalid instead of creating a new CMake file for it. Completely changing the build system to a library is a major change IMO

Reply:
Still think this is a very important change. @SSE4 @uilianries WDYT?

Reply:
I investigated librhash yesterday, the project is simple in terms of build. I don't like keeping a custom cmake file here if we can invoke autotools and msbuild, so I would prefer dropping any invalid configuration, if we can't use MSBuild. However, I opened a PR to author (https://github.com/rhash/RHash/pull/103), offering CMake support, which can be used by any user, including in our recipes. In summary, use autotools and msbuild for now.

Reply:
ping @madebr

Reply:
@uilianries  There is already a cmake pr rhash/RHash#71
I don't see them accepting a pr in the near future.